redis/src/pipeline.rs

```diff
@@ -293,6 +293,16 @@ macro_rules! implement_pipeline_commands {
                 self.ignored_commands.clear();
             }
 
+            /// Adds a prepared script command to the pipeline.
+            ///
+            /// Shorthand for `add_command(invocation.eval_cmd())`.
+            #[inline]
+            #[cfg(feature = "script")]
```

Comment:
instead of adding it here, can you add it to the `implement_commands` macro invocation in commands/mod.rs? that way it will be added to multiple relevant objets.

Reply:
Sounds, good. I didn't initially add it to `implement_commands` because it is not an actual Redis command.